db/migrate/20201012172713_remove_gpg_key_perms.rb

```diff
@@ -0,0 +1,22 @@
+class RemoveGpgKeyPerms < ActiveRecord::Migration[6.0]
+  def permission_map
+    {
+      Permission.find_by(name: :view_gpg_keys) => Permission.find_by(name: :view_content_credentials),
+      Permission.find_by(name: :edit_gpg_keys) => Permission.find_by(name: :edit_content_credentials),
+      Permission.find_by(name: :destroy_gpg_keys) => Permission.find_by(name: :destroy_content_credentials),
+      Permission.find_by(name: :create_gpg_keys) => Permission.find_by(name: :create_content_credentials)
+    }
+  end
+
+  def up
+    perms = permission_map
+    perms.each do |old_perm, new_perm|
+      Filtering.where(permission_id: old_perm.id).update_all(:permission_id => new_perm.id)
+    end
+    Permission.where(:name => perms.keys.map(&:id)).destroy_all
```

Comment:
name vs id mismatch

Reply:
doh! in my testing i checked the filterings but not the permission!

Reply:
<3 tests